src/hyp3_isce2/insar_tops.py

```diff
@@ -0,0 +1,120 @@
+"""Create a full SLC Sentinel-1 geocoded unwrapped interferogram using ISCE2's TOPS processing workflow"""
+
+import argparse
+import logging
+import os
+import site
+import sys
+from pathlib import Path
+from shutil import make_archive
+
+from hyp3lib.aws import upload_file_to_s3
+from hyp3lib.get_orb import downloadSentinelOrbitFile
+from hyp3lib.image import create_thumbnail
+
+from hyp3_isce2 import slc
+from hyp3_isce2 import topsapp
+from hyp3_isce2.dem import download_dem_for_isce2
+from hyp3_isce2.s1_auxcal import download_aux_cal
+
+
+log = logging.getLogger(__name__)
+
+# ISCE needs its applications to be on the system path.
+# See https://github.com/isce-framework/isce2#setup-your-environment
+ISCE_APPLICATIONS = Path(site.getsitepackages()[0]) / 'isce' / 'applications'
+if str(ISCE_APPLICATIONS) not in os.environ['PATH'].split(os.pathsep):
+    os.environ['PATH'] = str(ISCE_APPLICATIONS) + os.pathsep + os.environ['PATH']
+
+
+def insar_tops(
+    reference_scene: str,
+    secondary_scene: str,
+    swaths: list = [1, 2, 3],
+    polarization: str = 'VV',
+    azimuth_looks: int = 4,
+    range_looks: int = 20,
+) -> Path:
+    """Create a burst interferogram
```

Comment:
Should probably be "Create an interferogram" since this is a full SLC interferogram.

Reply:
Thanks for the catch!